wordpress-mobile/AztecEditor-Android
# Make sure code can be selected as the only current inline action and format

### Fix
https://github.com/wordpress-mobile/AztecEditor-Android/issues/967

This PR makes sure that the code action behaves as an exclusive action - it's the only action that can be selected at once. 

### Test
1. Click in an empty line with no text selected
2. Click on "Code" action
3. Notice it gets checked
4. Type some text
5. Notice it's styled with the "code" style
6. Click on "bold" action
7. Notice the "code" action is no longer checked
8. Notice the "bold" action is now checked
9. Type some text
10. Notice it's styled with the "bold" style


### Test
1. Select some text
2. Click on "code" style
3. Notice the text is styled in "code" style
4. Click on "italic" 
5. Notice the "code" style is removed and "italic" style is applied
6. Click on "code" style
7. Notice the "italic" style is removed and the "code" style is applied

Make sure strings will be translated:
